test/db2i.test.ts

```diff
@@ -16,6 +17,11 @@ describe('Db2iFormatter', () => {
 
   supportsComments(format, { nestedBlockComments: true });
   supportsCreateTable(format, { orReplace: true });
+  supportsAlterTable(format, {
+    addColumn: true,
+    dropColumn: true,
+    renameColumn: false,
```

Comment:
FYI: In general instead of specifying `false` here, better to just skip the option altogether. At least that's the style that the rest of the tests are following for now. Keeps the tests shorter, as like `supportsAlterTable()` takes an entire 5 boolean flags.

This concrete case I already cleaned up: https://github.com/sql-formatter-org/sql-formatter/commit/91e4be051c25df5e2cb29433e70495146b390b24

Reply:
Noted - only did it this way to clarify that it was on purpose in contrast to the DB2, which has `true` in `renameColumn`.